schemas/test/en/test_progress_value_source_blocks.json

```diff
@@ -0,0 +1,208 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "001",
+    "title": "Test progress value source",
+    "theme": "default",
+    "description": "A test survey for routing based on an number equals",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "section-1",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "s1-b1",
+                            "question": {
+                                "id": "s1-b1-q1",
+                                "title": "Section 1 Question 1",
+                                "description": [
+                                    "If you answer 0, then the second question of this first section will be skipped, and the second question of the second section will be displayed. If you answer anything else, then the second question of this first section will be displayed, and the second question of the second section will be skipped."
```

Comment:
Description is confusing for me. This example seem clearer to me but its personal preference:
`If you answer  0, then question two  from section one will be skipped and will display question two from section two. Any other answer will display question two from section one and will skip question two from section two.`

Reply:
Thanks, description wasn't even accurate. Replaced with 

```
If you answer 0, then the second question will be skipped because of a routing rule, as well as the fourth question because of a skip condition. Otherwise, question 2 and 4 will display.
```

How is that?

Reply:
Reads nicely 👍